Configuration fails to find libtiff-4 system library

### Describe the bug
<!--
    A clear and concise description of what the bug is.]
-->

`devtools::install()` complains that it cannot find `ibtiff-4` although `libtiff-dev` is installed on the system (Debian 12).

### To Reproduce
<!--
    Steps to reproduce the behavior.]
-->

GitHub Codespaces: https://codespaces.new/benz0li/unigd?hide_repo_select=true&ref=add-dev-container

```bash
radian
```

```R
devtools::install()
```

```
── R CMD build ──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
✔  checking for file ‘/workspaces/unigd/DESCRIPTION’ ...
─  preparing ‘unigd’:
✔  checking DESCRIPTION meta-information ...
─  cleaning src
─  running ‘cleanup’
─  checking for LF line-endings in source and make files and shell scripts
─  checking for empty or unneeded directories
─  building ‘unigd_0.1.0.tar.gz’
   Warning: file 'unigd/cleanup' did not have execute permissions: corrected
   Warning: file 'unigd/configure' did not have execute permissions: corrected
   
Running /usr/local/lib/R/bin/R CMD INSTALL /tmp/Rtmp8xVDte/unigd_0.1.0.tar.gz --install-tests 
* installing to library ‘/home/vscode/R/x86_64-pc-linux-gnu-library/4.3’
* installing *source* package ‘unigd’ ...
** using staged installation
Found pkg-config cflags and libs!
Using PKG_CFLAGS=-I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/freetype2 -I/usr/include/libpng16 
Using PKG_LIBS=-lcairo -lfreetype 
Using PKG_LIBTIFF_CFLAGS=-I/usr/include/x86_64-linux-gnu 
Using PKG_LIBTIFF_LIBS=-ltiff  -ltiffxx 
------------------------------[ WARNING ]---------------------------
Info: Configuration failed to find libtiff-4 system library.
httpgd is fully functional without libtiff-4, but some plot file formats
will not be available.
If you want to install libtiff-4 see below error for troubleshooting.
------------------------- ANTICONF ERROR ---------------------------
Configuration failed because libtiff-4 was not found. Try installing:
 * deb: libtiff-dev (Debian, Ubuntu, etc)
 * rpm: libtiff-devel (Fedora, EPEL)
 * brew: libtiff (OSX)
If libtiff-4 is already installed, check that 'pkg-config' is in your
PATH and PKG_CONFIG_PATH contains a libtiff-4.pc file. If pkg-config
is unavailable you can set INCLUDE_DIR and LIB_DIR manually via:
R CMD INSTALL --configure-vars='INCLUDE_DIR=... LIB_DIR=...'
--------------------------------------------------------------------
** libs
...
```

### Expected behavior
<!--
    A clear and concise description of what you expected to happen.]
-->

`configure` should be able to find the TIFF library files.

### Screenshots
<!--
    If applicable, add screenshots to help explain your problem.]
-->

\-

### Environment
 - OS: Debian 12
 - Architectrue: x86_64
 - R version: 4.3.2
 - unigd version: 0.1.0

### Additional context
<!--
    Add any other context about the problem here.]
-->

\-

> Thanks! This is also tracked in [nx10/httpgd#130](https://github.com/nx10/httpgd/issues/130)

OK

---

libtiff test configuration does not work as expected:

*configure.log*

```
src/sysdep_tests/sysdep_libtiff.cpp: In function ‘int main()’:
src/sysdep_tests/sysdep_libtiff.cpp:5:11: warning: unused variable ‘tiff’ [-Wunused-variable]
    5 |     TIFF *tiff = TIFFStreamOpen("memory", &tiff_ostream);
      |           ^~~~
/usr/bin/ld: /tmp/cc49Gs4W.o: in function `main':
/workspaces/unigd/src/sysdep_tests/sysdep_libtiff.cpp:5: undefined reference to `TIFFStreamOpen(char const*, std::ostream*)'
collect2: error: ld returned 1 exit status
```

--- a/configure
+++ b/configure
@@ -114,7 +114,7 @@ CXXFLAGS=`${R_HOME}/bin/R CMD config CXXFLAGS`
 CPPFLAGS=`${R_HOME}/bin/R CMD config CPPFLAGS`
 
 # Test configuration
-${CXX} ${CPPFLAGS} ${PKG_LIBTIFF_CFLAGS} ${CFLAGS} ${PKG_LIBTIFF_TEST_FILE} -o tmp_libtiff_test >/dev/null 2>configure.log
+${CXX} ${CPPFLAGS} ${PKG_LIBTIFF_CFLAGS} ${CFLAGS} ${PKG_LIBTIFF_TEST_FILE} ${PKG_LIBTIFF_LIBS} -o tmp_libtiff_test >/dev/null 2>configure.log
 # Customize the error
 if [ $? -ne 0 ]; then
   echo "------------------------------[ WARNING ]---------------------------"

